src/tracks/SampleTrack.cpp

```diff
@@ -408,6 +422,9 @@ void SampleTCOView::contextMenuEvent( QContextMenuEvent * _cme )
 				tr( "Set/clear record" ),
 						m_tco, SLOT( toggleRecord() ) );*/
 
+	contextMenu.addAction(embed::getIconPixmap("flip_x"),
+			tr("Reverse sample"), this, SLOT(reverseSample()));
```

Comment:
```suggestion
	contextMenu.addAction(
		embed::getIconPixmap("flip_x"),
		tr("Reverse sample"),
		this,
		SLOT(reverseSample()));
```
alternatively (my preference)
```suggestion
	contextMenu.addAction(
		embed::getIconPixmap("flip_x"),
		tr("Reverse sample"),
		this,
		SLOT(reverseSample())
	);
```